src/replicache.ts

```diff
@@ -715,8 +715,10 @@ export class Replicache<MD extends MutatorDefs = {}> {
 
       if (e instanceof PushError || e instanceof PullError) {
         online = false;
+        this._logger.error?.(`${name} threw:\n`, e, '\nwith cause:\n', e.cause);
```

Comment:
Needs a browser test too or we will print the cause twice

Reply:
This is with Firefox:

![image](https://user-images.githubusercontent.com/45845/142683147-13ade328-f614-4da1-97dc-08722828b8ee.png)

Reply:
which browser does this I tested Chrome, Safari and Firefox on my mac and none of them printed the cause.

Reply:
browser checks would be ugly, to avoid this whole issue i just changed the field name from `cause` to `causedBy`, so we avoid the inconsistent behavior while browser adoption of the new standard is influx.